localStorage.js

```diff
@@ -0,0 +1,17 @@
+// This Code will preserve data in the browser
+
+const contactForm = document.forms[0];
+const formValues = {};
+if (window.localStorage.getItem('show data')) {
+    const formObject = JSON.parse(window.localStorage.getItem('show data'));
+    contactForm.name.value = formObject.name;
+    contactForm.email.value = formObject.email;
+    contactForm.message.value = formObject.message;
+}
```

Comment:
Great job team 🤝

- [ ] [Optional] Data is not persisting in the name field after reload. You may kindly fix that as well.
![name](https://user-images.githubusercontent.com/63917725/180593129-f936573b-cf31-4785-a5a6-e66c630bee6c.PNG)